### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54875]
#### VFS team name
Authenticated Experience

#### VFS product name
My VA

#### VFS feature name
Audit UX Improvements

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
In the Healthcare section, two links (1. Request Travel Reimbursement and 2. Schedule and manage your appointments) send the user to other VA.gov pages that open in a new tab. Per the design system, they should open in the same window unless progress or data would be lost in doing so. 

Accessibility note: Opening in a new tab or window without warning can be particularly disruptive for screen reader users.

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Update the links to open within the same window

If opening in the same window isn't possible, add text indicating that the link opens in a new tab. Visible text is best, an `aria-label` would be minimally sufficient.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by sterkenburgsara** [comment]
@ajakabcin @aagosto90 hello! we have an update on this ticket. 

After much conversation with OCTO leads, it has been determined that subdomains (like MHV pages) are considered internal to VA.gov and should be opened within the same window, **not** in a new tab. This negates feedback we gave you last Thursday at your staging review (my apologies). All links on this entire page should open within the window. Since they shouldn't open in a new tab, they won't need the accessibility modifications to announce that to a screenreader. // cc @briandeconinck @mnorthuis @shiragoodman

**Comment by ajakabcin** [comment]
Hi @sterkenburgsara - thank you for this! We'll prioritize this in coming sprints

**Comment by ajakabcin** [comment]
Team will address this in #55580

**Comment by ajakabcin** [comment]
this has been fixed via #55580 - closing!
